rules.tf

```diff
@@ -420,7 +420,7 @@ resource "aws_wafv2_web_acl" "default" {
           for_each = lookup(rule.value, "statement", null) != null ? [rule.value.statement] : []
 
           content {
-            arn = ip_set_reference_statement.value.arn
+            arn = aws_wafv2_ip_set.default[local.ip_rule_to_ip_set[rule.key]]
```

Comment:
This doesnt work, it tries to evaluate the `aws_wafv2_ip_set.default` at plan time before the resource is created, unsure why